go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/foxboron/sbctl
 
-go 1.14
+go 1.15
```

Comment:
I'm not sure what the effect of increasing this is...

Reply:
I agree, will revert.